pco/src/latent_page_decompressor.rs

```diff
@@ -153,7 +153,7 @@ impl<L: Latent> LatentPageDecompressor<L> {
   }
 
   #[inline(never)]
-  unsafe fn decompress_offsets<const MAX_U64S: usize>(
+  unsafe fn decompress_offsets<const MAX_BYTES: usize>(
```

Comment:
What if we make an enum like
```
#[repr(u8)]
enum ReadWidth {
  U32,
  U64,
  U64x2,
}
```
and generally match on this thing, but use like `decompress_offsets::<read_width as u8>(...)` when it's needed in generic parameters?